main.go

```diff
@@ -43,7 +48,7 @@ func main() {
 	// set up signals so we handle the first shutdown signal gracefully
 	stopCh := signals.SetupSignalHandler()
 
-	cfg, err := clientcmd.BuildConfigFromFlags(masterURL, kubeconfig)
+	cfg, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
```

Comment:
I believe it is okay to pass nothing instead of `masterURL`, right?

Reply:
> I believe it is okay to pass nothing instead of masterURL, right?

Should be.  Make yourself a followup issues to auto-use an in cluster config by default.

Reply:
@deads2k It already works in-cluster. I'm not sure what package provides that feature, but I deployed it using the deployment manifest from #3, and it works. Are there any more details I should pay attention to?

Reply:
Weird, but if it works we won't mess with it for now.